INSTALL

```diff
@@ -109,7 +109,7 @@ Prepare your system
  2. Install dtrace (to generate provider.h)
     sudo apt-get install systemtap-sdt-dev
  3. Install libdispatch pre-reqs
-    sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev
+    sudo apt-get install libblocksruntime-dev libbsd-dev
```

Comment:
@MadCoder that makes sense. Once we have a final list of fixes needed and know which level of libkqueue those fixes make it in to, we can definitely take the approach of only using a submodule if the system installed levels are too old.